jsdojo/3-5/index.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width,initial-scale=1">
+<meta http-equiv="x-ua-compatible" content="IE=edge">
+<title>template</title>
+<link href="../../_common/css/style.css" rel="stylesheet" type="text/css">
+</head>
+<body>
+<header>
+<div class="header-contents">
+<h1>タイトル</h1>
+<h2>サブタイトル</h2>
+</div><!-- /.header-contents -->
+</header>
+<div class="main-wrapper">
+<section>
+	<div id ="g">
```

Comment:
- Remove spaces next to = so that this HTML has its consistency.
  All the HTML attributes in this file are described as <div class="classname">, which doesn't include spaces next to =.
- Format this element like `<div id="g"></div>`. It doesn't have to include extra spaces or line breaks.